client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-hosting-details.tsx

```diff
@@ -59,7 +59,16 @@ export const UpgradePlanHostingDetails: React.FC< Props > = ( {
 					<ul>{ hostingDetailsItems }</ul>
 				</div>
 				<div className="import__upgrade-plan-hosting-details-testimonials-container">
-					<p>{ translate( '100% loved by our best customers' ) }</p>
```

Comment:
@agrullon95 can you help me test this? I cannot seem to be able to reach the step/location where this renders. I looked through a recent PR of yours (https://github.com/Automattic/wp-calypso/pull/89497) but no luck. 🙏

Reply:
Hi @chriskmnds! 👋

I would recommend you to check the flows to see if this step is still used, or we just forgot to remove it in a refactor. Trying to navigate quickly, I wasn't able to see the step either.

This is the pricing step I see now:

<img width="1224" alt="Screenshot 2025-03-06 at 15 39 46" src="https://github.com/user-attachments/assets/20ef43c1-4169-4832-ac39-5e8b28fd262e" />

When we worked on that, we had a step similar to this one to present the pricing:

![plan page](https://github.com/user-attachments/assets/14aadf9f-db5a-41c0-b07b-705e24680941)

For more context, I found a PR where I worked on that old version: https://github.com/Automattic/wp-calypso/pull/90966